cli/js/mux_async_iterator.ts

```diff
@@ -0,0 +1,63 @@
+// Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
+
+/** Returns an async iterator which merges the given async iterators. */
+export function mux<T>(
```

Comment:
This seems quite similar to https://github.com/denoland/deno/blob/f4cc08c4fc97f86d82b860ac11a134dad75dfdca/std/util/async.ts#L34

If we're going to have this in core, maybe we should expose this as Deno.mux ?